packages/electron/src/preload-localStore.ts

```diff
@@ -43,6 +43,7 @@ const createLocalStore = () =>
       share: false,
       lastConfirmVersion: "",
       notification: true,
+      newVersionNotification: true,
```

Comment:
should this be false by default ?

Reply:
Hi Andrew, no, this is to track user's action. The banner will pop up when newVersionNotification and outdatedVersionFlag both are true (line 54 in Dashboard.tsx). By default, the banner will not pop up (line 18 in Dashboard.tsx). Thanks!